playbooks/installation/installation.adoc

```diff
@@ -324,13 +355,171 @@ In many cloud environments, it's common for you to assign hostnames to your host
 master.ose.example.com openshift_hostname=master.ose.example.com openshift_public_ip=10.3.4.5
 ----
 
+==== Smart Start/HA Example Host File
+This host file contains a core set of options to get your HA build up and running with a local HA Proxy to mimic a load balancer. 
+----
+# Create an OSEv3 group that contains the master, nodes, etcd, and lb groups.
+# The lb group lets Ansible configure HAProxy as the load balancing solution.
+# Comment lb out if your load balancer is pre-configured.
+[OSEv3:children]
+masters
+nodes
+etcd
+lb
+
+# Set variables common for all OSEv3 hosts
+[OSEv3:vars]
+ansible_ssh_user=root
+deployment_type=openshift-enterprise
+
+# Uncomment the following to enable htpasswd authentication; defaults to
+# DenyAllPasswordIdentityProvider.
+#openshift_master_identity_providers=[{'name': 'htpasswd_auth', 'login': 'true', 'challenge': 'true', 'kind': 'HTPasswdPasswordIdentityProvider', 'filename': '/etc/origin/master/htpasswd'}]
+
+# Native high availbility cluster method with optional load balancer.
+# If no lb group is defined installer assumes that a load balancer has
+# been preconfigured. For installation the value of
+# openshift_master_cluster_hostname must resolve to the load balancer
+# or to one or all of the masters defined in the inventory if no load
+# balancer is present.
+openshift_master_cluster_method=native
+openshift_master_cluster_hostname=openshift-cluster.example.com
+openshift_master_cluster_public_hostname=openshift-cluster.example.com
+
+# override the default controller lease ttl
+#osm_controller_lease_ttl=30
+
+# host group for masters
+[masters]
+master1.example.com
+master2.example.com
+master3.example.com
+
+# host group for etcd
+[etcd]
+etcd1.example.com
+etcd2.example.com
+etcd3.example.com
```

Comment:
the suggested smart start is 3 masters + 2 infra (nodes) +  3 app (nodes) and etcd is installed on the masters. or is the recommendation to split out etcd on its own